src/expressions/operators/compares/generalCompare.ts

```diff
@@ -157,3 +181,29 @@ export function generatePrefabFunction(
 		return result.getEffectiveBooleanValue();
 	};
 }
+
+function generateSingleGeneralCompareFunction(
```

Comment:
It feels a bit unnecessary to have two functions, one for single and one for multiple but if they can't be combined, this is fine.

Reply:
They can't as the singlecompare does a lot less